Makefile

```diff
@@ -561,15 +562,16 @@ gen-json = tools/doc/generate.js --format=json $< > $@
 gen-html = tools/doc/generate.js --node-version=$(FULLVERSION) --format=html \
 			--template=doc/template.html --analytics=$(DOCS_ANALYTICS) $< > $@
 
-gen-doc =	\
+install-yaml:
```

Comment:
Please add `install-yaml` to the list of phony targets.

Reply:
does `doc-targets` need to be added as well?